src/pages/getting-started/livechat-apps/index.mdx

```diff
@@ -46,6 +42,127 @@ See a 4-minute tutorial explaining how to create a simple app in the Developer C
+
+### **Step 1 Getting started**
+
+Make sure you are signed in to the [Developer Console](https://developers.livechat.com/console/). Your LiveChat credentials should allow you to be logged in automatically. If not, you can [sign up here](https://accounts.livechat.com/signup?client_id=bb9e5b2f1ab480e4a715977b7b1b4279&redirect_uri=https%3A%2F%2Fmy.livechatinc.com&response_type=token&source_id=create_account_link&source_url=https%3A%2F%2Faccounts.livechat.com&source_type=accounts).
+
+Now, we go to the **Apps** tab and click the **Create new app +** button. Give your app a name and click the **Continue** button. You can always change your name later.
+
+💡 **Need helping hand?** Right after signing in to the Developers Console, you can launch the **App wizard**. It will guide you through the whole app creation process.
+
+<img
+  alt="Creating LiveChat apps step1"
+  src="/images/getting-started/livechat-apps/LiveChat-apps-step1.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 2 Adding Agent App Widgets block**
+
+You can configure your app using Building blocks which are components that form your app. Depending on which building block you add to your app, the app will work differently. If you want to know how each building block can enrich your app, click the **Read docs** redirect and learn more about them. 
+
+To create a simple frontend app, we'll use the **Agent App Widgets building block**. 
+Agent App Widgets are web apps loaded inside the LiveChat App. They can interact and communicate with the LiveChat App. 
+
+<img
+  alt="Creating LiveChat apps step2"
+  src="/images/getting-started/livechat-apps/LiveChat -apps-step2.png"
+  width="1515px"
+  height="819px"
+/>
+
+To add a new Agent App Widget to your application, you need to follow these steps:
+1. Go to the **Building blocks** tab. 
+2. Add a new Agent App Widgets block by clicking the **Get started** button. 
+3. Enter the **Widget source URL**, which is the address of your application. If it's not ready yet, you can use our sample apps to play around.
+4. Now, go to the **Widget placement** and choose one. It allows you to decide where exactly in the LiveChat App you want to display your integration. We have chosen Full screen application in our example.
+5. The **Initial state** section is not required and works only for the Chat Details widget placement. It allows you to define the initial state for the Customer Details Widget. If you decide to use it, provide the state in the JSON format.
+6. Click the **Continue button** and enjoy your new app.
+
+<SectionLink to={"/getting-started/app-guides/agent-app-widgets#building-block-configuration"}>
+  Take a look at guidelines on how to fill in Building blocks details.
+</SectionLink>
+
+<img
+  alt="Creating LiveChat apps step2.1"
+  src="/images/getting-started/livechat-apps/LiveChat-appsstep2.1.1.png"
```

Comment:
```suggestion
  src="/images/getting-started/livechat-apps/LiveChat-apps-step2.1.1.png"
```

Reply:
file name changed > this would need to be altered in the local branch separately, I explained it in more detail somewhere below